src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -110,6 +110,10 @@ class ElasticSearchDAO(servers: Seq[Authority], indexName: String) extends Searc
     autocompleteSuggestions(client, indexName, criteria, groups)
   }
 
+  override def fieldSuggest(field: String, text: String): Future[Seq[String]] = {
```

Comment:
For my own edification - I was curious about the override here. Strictly, it's not necessary, but good practice to explicitly state it to prevent accidental override errors.